pkg/log/zap/zap.go

```diff
@@ -207,3 +208,44 @@ func NewRaw(opts ...Opts) *zap.Logger {
 	log = log.WithOptions(o.ZapOpts...)
 	return log
 }
+
+// BindToFlagSet func allows CLI passed flags to be parsed.
```

Comment:
```suggestion
// BindFlagSet will parse the given flagset for zap option flags and set the log options accordingly 
```
Also for the godoc comment here, can you add the list of flag names that BindFlags currently parses.